projects/cdk/package.json

```diff
@@ -15,12 +15,11 @@
     "repository": "https://github.com/tinkoff/taiga-ui",
     "license": "Apache-2.0",
     "dependencies": {
-        "@ng-web-apis/common": "^1.12.1",
-        "@ng-web-apis/mutation-observer": "^1.1.0",
-        "@ng-web-apis/resize-observer": "^1.0.3",
-        "@tinkoff/ng-event-plugins": "^2.5.1",
-        "@tinkoff/ng-polymorpheus": "^3.1.12",
-        "@types/resize-observer-browser": "^0.1.7",
+        "@ng-web-apis/common": "2.0.1",
+        "@ng-web-apis/mutation-observer": "2.0.0",
+        "@ng-web-apis/resize-observer": "2.0.0",
+        "@tinkoff/ng-event-plugins": "3.0.0",
+        "@tinkoff/ng-polymorpheus": "4.0.4",
```

Comment:
Because this is actual dependencies with fixed version that are bundled with cdk, and the rest are peer deps and they just need to work with 4 and above, they don't need to know about particular bugfixed.